backend/frontend/templates/frontend/user/board/partial_card.html

```diff
@@ -1,170 +1,225 @@
-                        <i class="fa-regular fa-face-frown fa-lg"></i>
-                    </div>
-                    <div class="font-bold bg-red-600 text-white rounded-full flex justify-center items-center h-5 px-2 text-xs -translate-x-1/2 -translate-y-2/4 top-0 right-0 mb-0.8 ml-1">{{card.code_review_ny_competent_since_last_review_request}} 
-                    </div>
+             {% endif %}
+             {% if card.code_review_ny_competent_since_last_review_request %}
+                <div class="relative inline-flex items-center p-2 text-sm font-medium text-center bg-opacity-10 rounded-lg">
+                  <i class="fa fa-frown-o" aria-hidden="true" title="Not yet competent reviews"></i>
+                  <div class="absolute inline-flex items-center justify-center w-5 h-5 text-xs text-white bg-red-600 border-2 border-white rounded-full -top-2 -end-2">{{ card.code_review_ny_competent_since_last_review_request }}</div>
                 </div>
-            {% endif %}
-            {% if card.code_review_red_flag_since_last_review_request %}
-                <div class="flex" title="Number of competent code reviews since last review request">
-                    <div>
-                        <i class="fa-regular fa-face-sad-tear fa-lg"></i>
-                    </div>
-                    <div class="font-bold bg-red-600 text-white rounded-full flex justify-center items-center h-5 px-2 text-xs -translate-x-1/2 -translate-y-2/4 top-0 right-0 mb-0.8 ml-1">{{card.code_review_red_flag_since_last_review_request}} 
-                    </div>
-                </div>    
-            {% endif %}
-        </div>
-    {% endif %}
+             {% endif %}
+             {% if card.code_review_red_flag_since_last_review_request %}
+                <div class="relative inline-flex items-center p-2 text-sm font-medium text-center bg-opacity-10 rounded-lg">
+                  <i class="fa-regular fa-face-sad-tear" aria-hidden="true" title="Red-flag reviews"></i>
+                  <div class="absolute inline-flex items-center justify-center w-5 h-5 text-xs text-white bg-red-600 border-2 border-white rounded-full -top-2 -end-2">{{ card.code_review_red_flag_since_last_review_request }}</div>
+                </div>   
+             {% endif %}
+             {% if card.code_review_excellent_since_last_review_request %}
+                <div class="relative inline-flex items-center p-2 text-sm font-medium text-center bg-opacity-10 rounded-lg">
+                  <i class="fa-regular fa-face-laugh-wink" aria-hidden="true" title="Excellent reviews"></i>
+                  <div class="absolute inline-flex items-center justify-center w-5 h-5 text-xs text-white bg-red-600 border-2 border-white rounded-full -top-2 -end-2">{{ card.code_review_excellent_since_last_review_request }}</div>
+                </div>
+             {% endif %}
+         </div>
+
+        {% if card.open_pr_count %}
+         <a href={{ card.repo_url }} title="Pull request info">
```

Comment:
There was a request to be able to navigate straight to the repo from the card even if there are no open PRs. This request was made with Tilde1, might as well implement it now.

Reply:
@Sbonelo01 The current workaround for this issue only works when there are open PRs. The previous card had a "Repository" text to achieve this. I don't know what needs to happen because the card design you are following does not consider this.

I think a GitHub icon that links to the repo if it exist should be fine for now, maybe there's a better way to do it.